phpstan-baseline.neon

```diff
@@ -5080,3 +5080,7 @@ parameters:
 			count: 1
 			path: src/Installer/OptionalModuleManager.php
 
+		-
```

Comment:
@nechai If we move config override to the `social_content_translation` we will not need this rule anymore.